backend/app.py

```diff
@@ -15,17 +16,20 @@
 			template_folder = "../dist")
 cors = CORS(app, resources={r"/api/*": {"origins": "*"}})
 
-DB = 1 # 1 = AsterixDB, 0 = Couchbase
-
-@app.route('/', defaults={'path': ''})
-@app.route('/<path:path>')
-def catch_all(path):
-	return render_template("index.html")
+#@app.route('/', defaults={'path': ''})
```

Comment:
I believe this would break the front-end by not serving it.

Reply:
Oh ok. I don't completely understand what it does. I commented it out because for some reason the "/api/search" and "/api/subscribe" stopped working, and commenting this out fixed it.
How is the front-end using it?

Reply:
It serves the front-end code that's been built to the client. Did you run `npm run build` to build the front-end prior to running it? Otherwise, you might not have the `dist` folder as it is excluded in the `.gitignore`.

Reply:
Nevermind. It works now.

Reply:
Awesome, thanks!